GetIntoTeachingApiTests/Models/TeacherTrainingAdviser/Validators/TeacherTrainingAdviserSignUpValidatorTests.cs

```diff
@@ -116,16 +118,29 @@ public class ReturningToTeacherTraining
         {
             private readonly TeacherTrainingAdviserSignUpValidator _validator;
             private readonly TeacherTrainingAdviserSignUp _request;
+            private readonly Mock<IStore> _mockStore =  new Mock<IStore>();
 
             public ReturningToTeacherTraining()
             {
-                _validator = new TeacherTrainingAdviserSignUpValidator(new Mock<IStore>().Object, new DateTimeProvider());
+                _validator = new TeacherTrainingAdviserSignUpValidator(_mockStore.Object, new DateTimeProvider());
                 _request = new TeacherTrainingAdviserSignUp() { TypeId = (int)Candidate.Type.ReturningToTeacherTraining };
             }
 
             [Fact]
             public void Validate_WhenValid_HasNoErrors()
             {
+                List<PickListItem> fakePickList = FakePickListItem.Default.Generate(100);
+                _mockStore.Setup(p => p.GetPickListItems(It.IsAny<string>(), It.IsAny<string>()))
+                    .Returns(fakePickList.AsQueryable);
+                PickListItem fakeCitizenship = new PickListItem { Id = 222750001, EntityName = "contact", AttributeName = "dfe_citizenship" };
+                PickListItem fakeSituation = new PickListItem() { Id = 12909, EntityName = "contact", AttributeName = "dfe_situation" };
+                PickListItem fakeVisastatus = new PickListItem() { Id = 12345, EntityName = "contact", AttributeName = "dfe_visastatus" };
+
+
+                fakePickList.Add(fakeCitizenship);
+                fakePickList.Add(fakeSituation);
+                fakePickList.Add(fakeVisastatus);
```

Comment:
I'm a bit confused by this setup: one pick-list has different _types_ of pick-list items? I think in reality we have a pick-list of citizenships, a pick-list of situations, a pick-list of visastatuses etc. Would it be better if the fakes closer resembled the real things?